### PR TITLE
Searchfeat

### PR DESCRIPTION
**Pull request by surbhi84**
this pr consists of:

- Search feature.
- added a loader.
- fixed empty pages.
- and some minor changes and fixes.

### CODE REVIEW

**Review by SJTGSHIVAM — APPROVED** [review]
Changes look good. We are good to merge this.

**Comment by SJTGSHIVAM** [comment]
I guess adding a _redirects in public folder might help

File: src/components/Searchbar.tsx [hunk 1]
```diff
@@ -1,15 +1,92 @@
...
+      setUsers(allUsers);
+    })();
+  }, []);
+
+  function searchInputHandler(
+    searchQuery: string,
+    users: Array<userTypeRaw>,
+    setSearchResult: React.Dispatch<React.SetStateAction<userTypeRaw[]>>
+  ) {
+    // console.log(searchQuery, "searchQyery");
```
Comment (jalajcodes) [inline src/components/Searchbar.tsx:26]: remove unused comments

Comment (jalajcodes) [inline src/components/Searchbar.tsx:26]: you can remove unused code

**Review by jalajcodes — APPROVED** [review]
Looks good!

**Pull request merged**